### DISCUSSION TITLE
For the onboarding PR, is it possible to point to a custom config in the `.github` repo?

### DISCUSSION THREAD

**Discussion by paddyroddy** [renovatebot/renovate#25377] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

Placeholder value, please select the correct response from the dropdown

### Wanted end result.

I would like the onboarding PR to suggest the following
```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": ["github>UCL-MIRSG/.github//renovate/default-config.json"]
}
```
as I have stored my default config in a file called `default-config.json` in the `.github` repo https://github.com/UCL-MIRSG/.github/blob/main/renovate/default-config.json

### What you tried so far.

-

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
This works, but the filename must be `renovate-config.json` at the root of the `.github` repo. It will then populate `local>org/.github:renovate-config` automatically.

Alternatively you can use `default.json` within a `org/renovate-config` repo

**Reply by paddyroddy** [reply]
Ah, that's a bit of a shame, I quite like our current path of `UCL-MIRSG/.github//renovate/default-config.json` which is in keeping with https://docs.renovatebot.com/config-presets/#github. I assume this is related to #22212?
